packages/forms/frontend/sirius-components-forms/src/propertysections/ButtonPropertySection.tsx

```diff
@@ -162,6 +180,12 @@ export const ButtonPropertySection: PropertySectionComponent<GQLButton> = ({
         ) : null}
         {widget.buttonLabel}
       </Button>
+      {loading ? (
+        <>
+          <CircularProgress className={classes.loadingIndicator} size={`${theme.spacing(2)}`} />
+          <Backdrop className={classes.loadingBackdrop} open={loading}></Backdrop>
```

Comment:
I'll remove the backdrop since it feels weird and it's not really understandable given that it's invisible (if it's not invisible, it's blinking all the time)